platform/linux-generic/Makefile.am

```diff
@@ -26,13 +26,11 @@ odpapiplatinclude_HEADERS = \
 		  include/odp/api/plat/atomic_inlines.h \
 		  include/odp/api/plat/byteorder_inlines.h \
 		  include/odp/api/plat/byteorder_inlines_api.h \
-		  include/odp/api/plat/packet_flag_inlines.h \
-		  include/odp/api/plat/packet_flag_inlines_api.h \
+		  include/odp/api/plat/packet_flags_inlines.h \
 		  include/odp/api/plat/packet_inline_types.h \
 		  include/odp/api/plat/packet_inlines.h \
 		  include/odp/api/plat/packet_inlines_api.h \
-		  include/odp/api/plat/pktio_inlines.h \
-		  include/odp/api/plat/pktio_inlines_api.h \
+		  include/odp/api/plat/packet_io_inlines.h \
```

Comment:
The patch set would be easier to follow if file names would not be changed, or at least file name changes would be done in a separate patch. I would not change pktio file name as API uses odp_pktio_ prefix instead of packet_io, so packet_io should be changed to pktio - not the other way around.

Reply:
Nice idea, thanks.